.insomnia/Request/req_2b193752cad6487691cd5ce0aef044d5.yml

```diff
@@ -0,0 +1,87 @@
+  review's articles and apply a certain customization on it.
+
+
+  All the customization are done in a background job, so the return status will be `ok` which means that the customization job was enqueued successfully.
+
+
+  Params:
+
+
+  - `extra`: This is a dictionary that contains facet-based filter parameters, which are described below. Parameters ending with `[]` can contain multiple values.
+      - `article_ids`: Filter based on ids.
+      - `mode`: Filter based on inclusion decision. Values can be `undecided`, `maybe`, `included`  `excluded` or `conflict` if the review blind mode is currently off.
+      - `searches[]`: Filter using search ids.
+      - `highlights_1[]`: Filter based on inclusion highlight keyword.
+      - `highlights_2[]`: Filter based on exclusion highlight keyword.
+      - `language[]`: Filter based on article language.
+      - `keyphrases[]`: Filter based on associated keywords, i.e. values from the “topics” facet.
+      - `locations[]`: Filters based on location of publication.
+      - `journal[]`: Filters based on journal of publication.
+      - `authors[]`: Filter based on listed authors.
+      - `year[]`: Filter based on year of publication.
+      - `publication_types[]` : Filter based on publication type.
+      - `user_labels[]`: Filter based on inserted user labels.
+      - `exclusion_labels[]`: Filter based on inserted exclusion labels.
+      - `abstract_languages[]`: Filter based on the abstract languages.
+      - `fulltext_types[]`: Filter based on uploaded full-text types. Values can be `pr` for private, `pu` for public.
+      - `customized_by[]`: Filter based on decisions made by users by specifiying user ids. If omitted, defaults to all users currently in the review.
+      - `decision_at_least`: Filter based on minimum collaborator decisions.
+      - `decision_at_most`: Filter based on maximum collaborator decisions.
+      - `dedup_result`: Filter based on dedup results. If not specified, retrieves all articles except those with the `deleted` dedup result. Values can be:
+          - 0 = unresolved
+          - 1 = deleted
+          - 2 = not duplicates
+          - 3 = resolved
+  				- all = retrieves all articles irrespective of dedup results.
+      - `dedup_job_id`: Filter based on dedup job id.
+      - `dedup_result_cluster_id`: Filter based on dedup result cluster id.
+      - `exact_matches[]`: Filter based on the number of exact matches hits.
+      - `pico_participants[]`: Filter based on PICO population keywords.
+      - `pico_intervention[]`: Filter based on PICO intervention keywords.
+      - `pico_control[]`: Filter based on PICO control keywords.
+      - `pico_outcome[]`: Filter based on PICO outcome keywords.
+
+  		
+  - `plan`
+
+
+  A plan is a hash with multiple keys and values denoting types of customizations to be made to the article. Details on the various customization types and their keys and values are provided next.
```

Comment:
If this is a copy, just refer to the other endpoint.

Reply:
I prefer to document it in the same endpoint as it's not an exact copy.